modules/developer-cli-odo-managing-defile-registries.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/configuring-the-odo-cli.adoc
+
+:_content-type: REFERENCE
+[id="developer-cli-odo-managing-defile-registries_{context}"]
+= Managing Devfile registries
+
+`odo` uses the portable *Devfile* format to describe the components. `odo` can connect to various Devfile registries to download Devfiles for different languages and frameworks.
+
+You can connect to publicly available Devfile registries, or you can install your own link:https://github.com/devfile/registry-support[Devfile Registry].
+
+You can use the `odo preference <add/remove> registry` command to manage the registries used by `odo` to retrieve Devfile information.
```

Comment:
This seems like a command, like your sections **Adding a registry** and **Deleting a registry**

Could this section be documented like:

**Managing a registry**

To manage the registries used by `odo` to retrieve Devfile information, run the following command:
`odo preference <add/remove> registry`

Reply:
If I'm overthinking it, feel free to disregard :)

Reply:
This is lead-in sentence for the following 2 sections, so I don't think it needs to be a separate section,